src/main/java/bhs/devilbotz/utils/RobotConfig.java

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.wpi.first.wpilibj.Filesystem;
+import java.io.File;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/** Add your docs here. */
+public class RobotConfig {
+
+  private static JsonNode robotConfig;
+
+  private static final String robotUniqueIdDefault =
```

Comment:
Is this necessary? Could you just use the compId String as the default instead of defining the same variable twice?

Reply:
Parker, do you mean: `robotUniqueIdDefault = compId'?

Reply:
I just moved/copied this as-is from Robot.java.

Reply:
Functionally, it's all good.